concepts/05 UI Components/DataGrid/05 Accessibility/15 Keyboard Navigation.md

```diff
@@ -7,100 +11,148 @@ A user can use the following keys to interact with the DataGrid component:
     </tr>
     <tr>
         <td>&larr; &rarr; &uarr; &darr;</td>
-        <td>
-        <ul>
-            <li>Navigates through cells if no cell is in an editing state.</li>
-            <li>In the editing state, moves the cursor inside a cell or increases/decreases cell value (depending on the editor).</li>
-        </ul>
-        </td>
+        <td>Moves focus between cells.</td>
+    </tr>
+    <tr>
+        <td>Tab<br/>Shift + Tab</td>
+        <td>Moves focus forward/backward between all DataGrid items.</td>
+    </tr>
+    <tr>
+        <td>Ctrl + &uarr;<br/>Ctrl + &darr;</td>
+        <td>Moves focus between the component's areas, such as the cell area, column headers area, and group panel.</td>
     </tr>
     <tr>
         <td>Shift + Mouse Wheel</td>
         <td>Scrolls content left/right if horizontal scrolling is active.</td>
```

Comment:
Actually, this only works with native scrolling. It doesn't work with simulated scrolling. I think we can talk about this.